pkg/virt-controller/watch/vmi.go

```diff
@@ -1122,6 +1123,14 @@ func (c *VMIController) sync(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod,
 				}
 			}
 		}
+		if len(vmispec.NetworksToHotplug(vmi.Spec.Networks, vmi.Status.Interfaces)) > 0 {
+			if err := c.handleDynamicInterfaceRequests(vmi, pod); err != nil {
+				return &syncErrorImpl{
+					err:    fmt.Errorf("failed to hotplug network interfaces for vmi [%s/%s]: %w", vmi.GetNamespace(), vmi.GetName(), err),
```

Comment:
nit: Why do you need to use methods to fetch the namespace and name? They are directly available without the need of a method.

Reply:
Why?

Reply:
no good reason. Do you have a good one for preferring yours ? Can we agree its the same thing ?

anyway, this is a nit. Choosing to ignore it.